### PR TITLE
Preserve style attributes of span while importing text

### PR DESCRIPTION
**Pull request by onurpolattimur**
fixes: #1158

### CODE REVIEW

File: src/util/__tests__/stripStyleAttribute.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import stripStyleAttribute from '../stripStyleAttribute'
+
+it('only allowed properties', () => {
+  expect(stripStyleAttribute('color: red; font-weight: bold')).toBe('color: red;font-weight: bold;')
+})
```
Comment (raineorshine) [inline src/util/__tests__/stripStyleAttribute.ts:5]: This test demonstrates that `stripStyleAttribute` does not strip `color` and `font-weight`, but it does not test that it strips other properties. Include a non-whitelisted property that is stripped for a complete test.

File: src/util/stripStyleAttribute.ts [hunk 2]
```diff
@@ -0,0 +1,70 @@
+interface StyleProperty {
+  name: string,
+  value: string,
+}
+
+const blackColors = ['#000000', '#000', 'rgb(0,0,0)', 'rgba(0,0,0,1)', 'black']
```
Comment (raineorshine) [inline src/util/stripStyleAttribute.ts:6]: Also `rgb(0, 0, 0) `

File: src/util/stripStyleAttribute.ts [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    })
+}
+
+/** Parse and strip style attribute, returns stripped style attribute string. */
+const stripStyleAttribute = (style: string) => {
+  const styles = parseStyleString(style)
+  return styles.reduce((acc, property) => {
+    const styleProperty = allowedStyleProperties.find(allowedStyleProperty => property.name.startsWith(allowedStyleProperty.property))
+    if (styleProperty && (!styleProperty.test || styleProperty.test(property, styles))) {
+      acc += `${property.name}: ${property.value};`
```
Comment (raineorshine) [inline src/util/stripStyleAttribute.ts:64]: Please avoid mutations when possible

File: src/util/stripStyleAttribute.ts [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+      }
+    })
+}
+
+/** Parse and strip style attribute, returns stripped style attribute string. */
+const stripStyleAttribute = (style: string) => {
+  const styles = parseStyleString(style)
+  return styles.reduce((acc, property) => {
+    const styleProperty = allowedStyleProperties.find(allowedStyleProperty => property.name.startsWith(allowedStyleProperty.property))
+    if (styleProperty && (!styleProperty.test || styleProperty.test(property, styles))) {
```
Comment (raineorshine) [inline src/util/stripStyleAttribute.ts:63]: You'll want to `toLowerCase` the property.

**Review by raineorshine — CHANGES_REQUESTED** [review]
Thank you! Good structure, just a few small suggestions.

File: src/util/__tests__/stripStyleAttribute.ts (hunk 1 above)
Comment (onurpolattimur) [reply src/util/__tests__/stripStyleAttribute.ts:5]: There is a test like you said. The purpose of this test is to check the output for the allowed properties only.

**Review by raineorshine — APPROVED** [review]

**Pull request merged**